cmd/provisioner-localpv/app/config.go

```diff
@@ -118,6 +119,17 @@ func (p *Provisioner) GetVolumeConfig(pvName string, pvc *v1.PersistentVolumeCla
 	return c, nil
 }
 
+//GetSharedMountValue return the value of KeyShared value configured
+// in StorageClass. Default is false.
+func (c *VolumeConfig) GetSharedMountValue() bool {
+	sharedMountValue := c.getValue(KeyShared)
+	if strings.TrimSpace(sharedMountValue) == "true" {
```

Comment:
can use https://github.com/openebs/maya/blob/6daf5f0486e8f3bfcf263c6916e84b562fac0efc/pkg/util/util.go#L53 here, instead of manual comparison.